mesonbuild/modules/i18n.py

```diff
@@ -27,6 +36,9 @@ def gettext(self, state, args, kwargs):
         datadirs = mesonlib.stringlistify(kwargs.get('data_dirs', []))
         extra_args = mesonlib.stringlistify(kwargs.get('args', []))
 
+        if not self.has_gettext_tools():
+            mlog.warning("Some gettext tools were not found. Installation of languages won't be available until all tools are installed")
```

Comment:
Probably should only print this warning once similar to other warnings in the gnome module.